src/components/steps/StepAddManager.tsx

```diff
@@ -33,7 +35,25 @@ export function StepAddManager({
   const [people, setPeople, setPeopleHistory] = useHistoryState<UsersObject>('people_step2', {});
   const [peopleErrors, setPeopleErrors] = useState<UsersError>();
   const [genericError, setGenericError] = useState<boolean>(false);
+  const requestIdRef = useRef<string>();
```

Comment:
There is a reason why you using useRef in this case?

Reply:
I dont want to lose the value between component rerenders. The value should not change after rerender of component.